### ISSUE TITLE
Failed generate IOS app when execute command "flutter build ipa"

### ISSUE THREAD

**Issue by MustafaHasria** [flutter/flutter#81357]
Hi all, 
after I run flutter build ipa I got these errors 

```
Failed to build iOS app
Error output from Xcode build:
↳
    objc[29337]: Class AMSupportURLConnectionDelegate is implemented in both ?? (0x20b4d67a0) and ?? (0x117fc02b8). One of the two will be used. Which one is undefined.
    objc[29337]: Class AMSupportURLSession is implemented in both ?? (0x20b4d67f0) and ?? (0x117fc0308). One of the two will be used. Which one is undefined.
    ** ARCHIVE FAILED **


Xcode's output:
↳
    /Users/mustafahasria/Desktop/Flutter Projects/Projects/talentslist-app/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module 'connectivity' not found
    @import connectivity;
     ~~~~~~~^~~~~~~~~~~~
    1 error generated.
    /Users/mustafahasria/Desktop/Flutter Projects/Projects/talentslist-app/ios/Runner/GeneratedPluginRegistrant.m:10:9: fatal error: module 'connectivity' not found
    @import connectivity;
     ~~~~~~~^~~~~~~~~~~~
    1 error generated.
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Constructing build description

Encountered error while archiveing for device.

```

and these my flutter doctor -v 
```

[✓] Flutter (Channel stable, 2.0.4, on macOS 11.0.1 20B29 darwin-arm, locale en-SY)
    • Flutter version 2.0.4 at /Users/mustafahasria/flutter
    • Framework revision b1395592de (4 weeks ago), 2021-04-01 14:25:01 -0700
    • Engine revision 2dce47073a
    • Dart version 2.12.2

[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at /Users/mustafahasria/Library/Android/sdk
    • Platform android-30, build-tools 30.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 12.4, Build version 12D4e
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 4.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6915495)

[✓] VS Code (version 1.55.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.21.0

[✓] Connected device (2 available)
    • iPhone 12 Pro Max (mobile) • 4D008366-0EAF-4B70-BA88-5861FE89A087 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-14-4 (simulator)
    • Chrome (web)               • chrome                               • web-javascript • Google Chrome 90.0.4430.93

• No issues found!


```

**Comment by TahaTesser** [comment]
Hi @MustafaHasria 
Is this with just `flutter build ipa` or also with `flutter run -v`?
Please try deleting Podfile and Pods file and also Podspec file

Can you please upgrade to the latest  `stable` and try again
`flutter channel stable`
`flutter upgrade --force`
`flutter doctor -v` 

Related to https://github.com/flutter/flutter/issues/73554

**Comment by MustafaHasria** [comment]
> Hi @MustafaHasria
> Is this with just `flutter build ipa` or also with `flutter run -v`?
> Please try deleting Podfile and Pods file and also Podspec file
> 
> Can you please upgrade to the latest `stable` and try again
> `flutter channel stable`
> `flutter upgrade --force`
> `flutter doctor -v`
> 
> Related to #73554

i did what you said but i still has the same problem 
[✓] Flutter (Channel stable, 2.0.5, on macOS 11.0.1 20B29 darwin-arm, locale en-SY)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 4.1)
[✓] VS Code (version 1.55.2)
[✓] Connected device (1 available)

• No issues found!

**Comment by TahaTesser** [comment]
Hi @MustafaHasria 
Can you please provide a [minimal complete reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) and complete `flutter run --verbose` logs (paste the logs in a text file, since logs are too long to paste in the comment for iOS) running the minimal code sample?
Thank you

**Comment by pedromassangocode** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. 
We are therefore reluctantly going to close this bug for now. 
Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!  
Thanks for your contribution.

Could everyone who still has this problem please file a new issue with the exact description of what happens, logs, and the output of `flutter doctor -v`.
All system setups can be slightly different, so it's always better to open new issues and reference related issues.

**Comment by heinzan** [comment]
same me too
